shared/api/server.go

```diff
@@ -105,6 +105,32 @@ type ServerEnvironment struct {
 	// List of active storage driver versions (separate by " | ")
 	// Example: 1 | 0.8.4-1ubuntu11
 	StorageVersion string `json:"storage_version" yaml:"storage_version"`
+
+	// List of supported storage drivers
+	StorageSupportedDrivers []ServerStorageDriverInfo `json:"storage_supported_drivers" yaml:"storage_supported_drivers"`
+}
+
+// ServerStorageDriverInfo represents the read-only info about a storage driver
+//
+// API extension: server_supported_storage_drivers
```

Comment:
To avoid `API extension: ...` making it into the swagger model, you want to do:

```
// ServerStorageDriverInfo represents the read-only info about a storage driver
//
// swagger:model
//
// API extension: server_supported_storage_drivers
```

Reply:
OK will change to that, out of interest, why don't we want to indicate the API extension that the type becomes available from?

Reply:
Is it because swagger will only take the last line of the docblock for its description, and without that, it replaces the description?

Reply:
Yeah, I wouldn't mind it making it into swagger if there was a proper field for it, but it treating it as the description potentially overriding actual useful information is the issue :)

Reply:
Makes sense thanks.